ripley/models/admin_user.py

```diff
@@ -22,54 +22,40 @@
 "AS IS". REGENTS HAS NO OBLIGATION TO PROVIDE MAINTENANCE, SUPPORT, UPDATES,
 ENHANCEMENTS, OR MODIFICATIONS.
 """
+from datetime import datetime
+
 from ripley import db, std_commit
 from ripley.lib.util import to_isoformat
-from ripley.models.base import Base
 
 
-class UserAuth(Base):
-    __tablename__ = 'user_auths'
+class AdminUser(db.Model):
+    __tablename__ = 'admin_users'
 
-    id = db.Column(db.Integer, nullable=False, primary_key=True)  # noqa: A003
-    active = db.Column(db.Boolean, nullable=False, default=False)
+    uid = db.Column(db.String(255), nullable=False, primary_key=True)
     is_canvas_whitelisted = db.Column(db.Boolean, nullable=False, default=False)
```

Comment:
@johncrossman will there be a follow-up PR to remove this column if we're no longer making use of it?

Reply:
@pauline2k - Whoops. That was supposed to be removed after discuss w/ Miles. I'll amend this PR.